### PR TITLE
added implementation for different timestamp format

### PR DESCRIPTION
**Pull request by SambitKSahu99**
added new class for time stamp sorting and added sql scripts directory which contains all sql scripts

### CODE REVIEW

File: SqlScripts/Create AUIDT table sql script.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+use poc_1;
+create table audit(
+record_id int not null auto_increment primary key,
+logfile_folder_path varchar(100),
+total_files int,
+name_of_files varchar(500),
+date_time_of_operation varchar(45),
+result varchar (45),
+output_file_name varchar(45),
+error_message varchar(100));
```
Comment (vinodkadam22) [inline SqlScripts/Create AUIDT table sql script.sql:10]: Add a line at last

File: SqlScripts/Insert into AUDIT table sq script.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+use poc_1;
+insert into audit (logfile_folder_path,total_files,name_of_files,date_time_of_operation,result,output_file_name,error_message) values (?,?,?,?,?,?,?);
```
Comment (vinodkadam22) [inline SqlScripts/Insert into AUDIT table sq script.sql:2]: Add a line at last

File: src/main/java/org/logAggregatorTool/operation/TimeStampProcessor.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    public final List<SimpleDateFormat> POSSIBLE_DATE_FORMATS = Arrays.asList(new SimpleDateFormat(LogAggregatorToolConstants.POSSIBLE_DATE_FORMAT_1), new SimpleDateFormat(LogAggregatorToolConstants.POSSIBLE_DATE_FORMAT_2));
+
+    public Date parsingTimeStamp(String timeStamp) {
+        for (SimpleDateFormat possibleDateFormat : POSSIBLE_DATE_FORMATS) {
+            try {
+                return possibleDateFormat.parse(timeStamp);
+            } catch (ParseException parseException) {
+//                System.out.println(LogAggregatorToolConstants.PARSING_TIMESTAMP_ERROR);
+            }
+        }
```
Comment (vinodkadam22) [inline src/main/java/org/logAggregatorTool/operation/TimeStampProcessor.java:23]: Do not comment line 21, Try handling it

File: SqlScripts/Create AUIDT table sql script.sql (hunk 1 above)
Comment (SambitKSahu99) [reply SqlScripts/Create AUIDT table sql script.sql:10]: resolved

File: SqlScripts/Insert into AUDIT table sq script.sql (hunk 2 above)
Comment (SambitKSahu99) [reply SqlScripts/Insert into AUDIT table sq script.sql:2]: resolved

File: src/main/java/org/logAggregatorTool/operation/TimeStampProcessor.java (hunk 3 above)
Comment (SambitKSahu99) [reply src/main/java/org/logAggregatorTool/operation/TimeStampProcessor.java:23]: resolved

File: SqlScripts/Create AUIDT table sql script.sql
Comment (divya-sathyan) [inline SqlScripts/Create AUIDT table sql script.sql:1]: Check the audit spelling

**Review by ashutosh-2609 — APPROVED** [review]

**Review by divya-sathyan — APPROVED** [review]

**Review by vinodkadam22 — APPROVED** [review]

**Pull request merged**